media/js/uelc_admin/web_socks.js

```diff
@@ -87,23 +91,33 @@ $(function() {
         alert($('Your browser does not support WebSockets. ' +
                 'You will have to refresh your browser to view updates.'));
     }
+    var displayDecisionTitle = function(gcs, sectionBlock, data) {
+        gateSection = sectionBlock.find('.gate-section');
+        console.log(sectionBlock);
+        console.log(gateSection);
+        gateSection.append('<div class-"response">' +
+            data.notification.data + '</div>');
+    };
     var setGroupLocation = function(gcs, sectionBlock) {
         var groupIcon = jQuery('<span class="glyphicon glyphicon-user" ' +
             'aria-hidden="true"></span>');
         jQuery(gcs).find('.glyphicon-user').remove();
-        sectionBlock.find('.gate-button').prepend(groupIcon);
+        sectionBlock.find('.gate-section').append(groupIcon);
     };
 
     var updateGateSectionStatus = function(gcs, sectionBlock, action) {
         var badge = sectionBlock.find('.badge');
         if (action === 'section submitted' || action === 'made decision') {
             badge.text('reviewed');
+            badge.removeClass('reviewing').addClass('reviewed');
             return;
         }
         if (badge.text() === 'reviewed') {
             return;
         }else {
             badge.text('reviewing');
+            badge.removeClass('to').removeClass('be').removeClass('reviewed');
```

Comment:
It looks like you can give `removeClass` multiple classes separated by spaces, so this could be changed to `badge.removeClass('to be reviewed')`